code/XMLTester.xml

```diff
@@ -955,6 +955,7 @@
 </participant>
 
 <participant profile_name="part_builtin_example">
+<domainId>4</domainId> <!-- uint32 -->
```

Comment:
#435 removed the `<domainId>` tag from the `domainparticipant_profile_name` snippet and not from the `part_builtin_example` profile. Have you checked that the `<domainId>` tag is being shown in the snippet in the XML section?

Also, the tag is not indented.

Reply:
You were right. Doc built and tested locally successfully